### PR TITLE
FIX Init array, use $oldName, add REPOS_EXCLUDE, check if label exists before rename

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/154

Need to init the array to prevent an error if no repos were matched using --only / --exclude falgs

Use $oldName so that the correct URL is used when rewriting an old label

Add REPOS_EXCLUDE which has some repos that should never have their labels updated

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Some very small changes.
Also needs rebase.

File: labels_command.php [hunk 1]
```diff
@@ -93,17 +111,28 @@
             // https://docs.github.com/en/rest/issues/labels#update-a-label
             if (array_key_exists($name, LABELS_RENAME)) {
                 $newName = LABELS_RENAME[$name];
-                info("Updating label $name to $newName in $repo");
-                if ($input->getOption('dry-run')) {
-                    info('Not updating label on GitHub because --dry-run option is set');
-                } else {
-                    github_api($url, ['new_name' => $newName], 'PATCH');
+                // check if label already exists
```
[the remark this answers is not in the document]
Comment (emteknetnz) [reply labels_command.php]: Updated

File: labels_command.php [hunk 2]
```diff
@@ -40,6 +40,24 @@
...
+    'classproxy',
+    'installers',
+    'GridFiedBulkEditingTools',
+    'phockito',
+    'silverstripe-debugbar',
+    'silverstripe-elemental-subsites',
+    'silverstripe-fluent',
+    'silverstripe-phockito',
+    'silverstripe-proxy-db',
+    'sortablegridfield',
```
[the remark this answers is not in the document]
Comment (emteknetnz) [reply labels_command.php]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**